kpops/components/base_components/kubernetes_app.py

```diff
@@ -18,11 +18,12 @@
 )
 
 
-class KubernetesAppConfig(BaseModel):
-    """Settings specific to Kubernetes apps."""
+class KubernetesAppConfig(CamelCaseConfigModel, DescConfigModel):
+    """Settings specific to Kubernetes Apps."""
```

Comment:
I believe lowercase is correct here since it's pretty general for all Kubernetes objects, its only purpose is to enforce the camel casing